content/en/blog/_posts/2024-12-11-Kubernetes-v1-32-Release.md

```diff
@@ -0,0 +1,433 @@
+
+This work was done as a part of [KEP #3476](https://github.com/kubernetes/enhancements/issues/3476), by [SIG
+Storage](https://github.com/kubernetes/community/tree/master/sig-storage).
+
+### Structured parameter support
+
+The core part of Dynamic Resource Allocation (DRA), the structured parameter support, got promoted to beta.
+This allows the kube-scheduler and Cluster Autoscaler to simulate claim allocation directly, without needing a
+third-party driver.
+These components can now predict whether resource requests can be fulfilled based on the cluster's current state without actually
+committing to the allocation. By eliminating the need for a third-party driver to validate or test allocations, this
+feature improves planning and decision-making for resource distribution, making the scheduling and scaling processes
+more efficient.
+
+This work was done as a part of [KEP #4381](https://github.com/kubernetes/enhancements/issues/4381), by WG Device
+Management (a cross functional team containing [SIG Node](https://github.com/kubernetes/community/tree/master/sig-node),
+[SIG Scheduling](https://github.com/kubernetes/community/tree/master/sig-scheduling) and [SIG
+Autoscaling](https://github.com/kubernetes/community/tree/master/sig-autoscaling)).
+
+## Highlights of new features in Alpha
+
+_This is a selection of key improvements introduced as alpha features in the v1.32 release._
+    
+### Asynchronous preemption in the Kubernetes Scheduler
+
+The Kubernetes scheduler has been enhanced with Asynchronous Preemption, a feature that improves scheduling throughput
+by handling preemption operations asynchronously. Preemption ensures higher-priority pods get the resources they need by
+evicting lower-priority ones, but this process previously involved blocking operations like PostFilter execution,
+slowing down the scheduler. With this enhancement, such tasks are now processed in parallel, allowing the scheduler to
+continue scheduling other pods without delays.
+This improvement is particularly beneficial in clusters with high Pod churn or frequent scheduling failures, ensuring a
+more efficient and resilient scheduling process.
+
+This work was done as a part of KEP [#4832](https://github.com/kubernetes/enhancements/issues/4832) by [SIG
+Scheduling](https://github.com/kubernetes/community/tree/master/sig-scheduling).
+
+ 
+### Mutating admission policies using CEL expressions
+
+This feature leverages CEL's object instantiation and JSON Patch strategies, combined with Server Side Apply’s merge
+algorithms. It simplifies policy definition, reduces mutation conflicts, and enhances admission control performance
+while laying a foundation for more robust, extensible policy frameworks in Kubernetes.
+
+The Kubernetes scheduler now supports Common Expression Language (CEL)-based Mutating Admission Policies, providing a
```

Comment:
maybe:  scheduler -> apiserver

Reply:
Good point, but: “API server“ not “apiserver“

Reply:
> maybe: scheduler -> apiserver

+1, I was also confused when I saw this, both [KEP](https://github.com/kubernetes/enhancements/issues/3962) and [code implementation](https://github.com/kubernetes/kubernetes/pull/127134) are related to the `API server `component, not the scheduler component.

Reply:
Fixed, thanks for catching this!